mkt/reviewers/utils.py

```diff
@@ -31,15 +31,24 @@
 log = commonware.log.getLogger('z.mailer')
 
 
+def get_review_type(request, addon, version):
+    if EscalationQueue.objects.filter(addon=addon).exists():
+        queue = 'escalated'
+    elif RereviewQueue.objects.filter(addon=addon).exists():
+        queue = 'rereview'
+    else:
+        queue = 'pending'
+    return queue
+
```

Comment:
I see you've just moved this from ReviewBase, but how are apps in the updates queue handled?